packages/create-discord-bot/bin/index.ts

```diff
@@ -40,6 +40,7 @@ program
 	})
 	.option('--typescript', 'Whether to use the TypeScript template.')
 	.option('--javascript', 'Whether to use the JavaScript template.')
+	.option('--noInstall', 'Whether to not automatically install the packages.')
```

Comment:
Should this not be `--no-install`?

Reply:
Crawl used diffrent syntax so for continuity i also used it

Reply:
This (and that) should change.